frontend/packages/dev-console/src/components/pipelines/modals/triggers/TriggerBindingSection.scss

```diff
@@ -1,9 +1,15 @@
-.odc-add-trigger-form {
+.odc-trigger-binding-section {
```

Comment:
Hah, nice! I guess through refactors I forgot to update the css variable name. Good catch!